src/components/cc-logs-controller/cc-logs-controller.js

```diff
@@ -0,0 +1,504 @@
+          @cc-select:input=${this._onDateDisplayChange}
+        ></cc-select>
+
+        <cc-select
+          label="${i18n('cc-logs-controller.timezone')}"
+          .options=${TIMEZONE_CHOICES}
+          .value=${this.timezone}
+          @cc-select:input=${this._onTimezoneChange}
+        ></cc-select>
+      </div>
+    `;
+  }
+
+  _renderMetadataOptions () {
+    const metadataDisplayEntries = Object.entries(this.metadataDisplay ?? {});
+    if (metadataDisplayEntries.length === 0) {
+      return null;
+    }
+
+    return html`
+      <div class="options-header">
+        <cc-icon .icon=${metadataIcon} size="lg"></cc-icon>
+        <span>${i18n('cc-logs-controller.option-header.metadata')}</span>
+      </div>
+      <div class="options-group">
+        ${metadataDisplayEntries.map(([name, display]) => {
+          return html`
+            <label
+              for="metadata-${name}"
+              class=${classMap({ span: display.icon == null })}
+            >
+              <input id="metadata-${name}"
+                     type="checkbox"
+                     data-name="${name}"
+                     @change=${this._onMetadataChange}
+                     .checked=${!display.hidden}> ${display.label}
+            </label>
+          `;
+        })}
+      </div>
+      </div>
+    `;
+  }
+
+  static get styles () {
+    return [
+      // language=CSS
+      css`
+        :host {
+          display: grid;
+          align-items: center;
+          column-gap: 0.35em;
+          grid-template-areas: 
+            'header-slot scroll-button options-popover'
+            'logs        logs          logs';
+          grid-template-columns: 1fr auto auto;
+          grid-template-rows: max-content 1fr;
+          row-gap: 0.5em;
+        }
+
+        .header-slot {
+          grid-area: header-slot;
+        }
+
+        .header-scroll-button {
+          grid-area: scroll-button;
+        }
+
+        .header-options-popover {
+          grid-area: options-popover;
+        }
+
+        .logs {
+          align-self: normal;
+          grid-area: logs;
+        }
+
+        .options {
+          margin: 0.5em 1em;
+          white-space: nowrap;
+        }
+
+        .options-header {
+          display: grid;
+          width: 100%;
+          align-items: center;
+          color: var(--cc-color-text-weak, #333);
+          font-weight: bold;
+          grid-gap: 0.5em;
+          grid-template-columns: auto auto 1fr;
+        }
+
+        .options-header:not(:first-of-type) {
+          margin-top: 1.85em;
+        }
+
+        .options-header::after {
+          height: 1px;
+          background-color: var(--cc-color-text-weak, #333);
+          content: '';
+        }
+
+        .options-group {
+          display: grid;
+          margin: 1em 0.75em 0.75em;
+          grid-template-columns: minmax(max-content, 1fr);
+          row-gap: 0.75em;
+        }
+
+        input[type='checkbox'] {
+          margin: 0;
```

Comment:
**fix:** it looks like inputs do not scale when resizing the font so I suggest adding these to fix the issue (feel free to change the values of course)
```suggestion
          margin: 0;
          width: 1em;
          height: 1em;
```